src/java.base/share/classes/java/text/BreakIterator.java

```diff
@@ -99,6 +99,12 @@
  * and a diacritical mark. What users consider to be a character can
  * differ between languages.
  *
+ * @implNote The default implementation of the character boundary analysis
+ * conforms to the Unicode Consortium's Extended Grapheme Cluster breaks.
+ * For more detail, refer to
+ * <a href="https://www.unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries">
+ * Grapheme Cluster Boundaries</a> section in the Unicode Standard Annex #29.
+ *
```

Comment:
I would say this is an `@implSpec` instead of an `@implNote` because we want applications to be able to rely on this behavior if they're using the default implementation. Also please make corresponding update to the CSR.

Reply:
Fixed